python/runtime/db.py

```diff
@@ -370,3 +371,37 @@ def buffered_db_writer(driver,
         yield w
     finally:
         w.close()
+
+
+def get_table_schema(datasource, table):
```

Comment:
Can we pass a connection as an argument, so that we don't need to create a new connection every time we call this function?

Reply:
Done